tmeister/static/src/toggles/delete/delete-modal-metrics.component.js

```diff
@@ -11,10 +20,24 @@ export default function DeleteFeatureModal (props) {
         {
           Object.keys(metrics).map((key) => {
             const countTimes = sumBy(metrics[key], i => i.hit_count)
-            const dayCount = metrics[key].length
+            const dayArray = metrics[key].map(obj => dayjs(obj.date))
+              .sort((a, b) => {
+                if (a.isBefore(b)) {
+                  return -1
+                } else {
+                  return 1
+                }
+              })
+            const dayCount = get(metrics, `[${key}].length`, 0)
```

Comment:
huh -- `.length` works here, eh?

Reply:
Weird right? I wasn't sure if it would or not. I'm more and more convinced that `lodash` is actually sorcery :wink:

Reply:
I suppose it would...